src/views/contact/ContactPage.js

```diff
@@ -1,30 +1,40 @@
 import React from 'react';
+import styled from '@datapunt/asc-core';
+import { themeSpacing } from '@datapunt/asc-ui';
 
+import { Heading } from '@datapunt/asc-ui';
 import BlueLinkButton from 'shared/buttons/BlueLinkButton';
 import { appRoutes } from 'constants.js';
 import DataTable from '../../shared/dataTable/DataTable';
-import styles from './ContactPage.module.scss';
 import { ReactComponent as Chevron } from 'assets/icons/chevron-left.svg';
 import Footer from '../../components/footer/Footer';
+import ContactPageStyle, {
+  ContentStyle,
+  EmailLinkStyle,
+} from './ContactPageStyle';
+import { IntroTextStyle } from '../../styles/SharedStyles';
 
+const StyledHeading = styled(Heading)`
```

Comment:
What is the logic behind using imported styled components in other components, but using an inline styled component for the `ContactPage` component?

Reply:
moved to the styles module